src/import-v1/index.ts

```diff
@@ -548,6 +553,12 @@ export async function importV1TokenOfferingData(db: PgWriteStore) {
 }
 
 export async function handleBnsImport(db: PgWriteStore) {
+  const stacksNodeType = process.env.STACKS_NODE_TYPE;
```

Comment:
We should also add this to the `.env` file where we informally document env vars